src/compounds/Item/Item.jsx

```diff
@@ -65,6 +67,8 @@ const Item = ({ buttonLink, buttonProps, href, isLoading, item, orientation, tit
                 buttonLink={link}
                 buttonProps={buttonProps}
                 item={item}
+                markdownDescriptionTruncated={markdownDescriptionTruncated}
+                markdownDescriptionExtended={markdownDescriptionExtended}
```

Comment:
can these get added to the propType and defaultProps definitions too?

Reply:
done!